program_info/win_install.nsi.in

```diff
@@ -336,6 +348,19 @@ Section "" UninstallPrevious
 
 SectionEnd
 
+;------------------------------------
+; include nice plugins
+
+; NScurl - curl in NSIS
+; used for MSVS redist download
+; extract to ../NSISPlugins/NScurl
+; https://github.com/negrutiu/nsis-nscurl/releases/latest/download/NScurl.zip
+!insertmacro CompileTimeIfFileExist "../NSISPlugins/NScurl/Plugins/" haveNScurl
+!ifdef haveNScurl
+!AddPluginDir /x86-unicode     "../NSISPlugins/NScurl/Plugins/x86-unicode"
+!AddPluginDir /x86-ansi        "../NSISPlugins/NScurl/Plugins/x86-ansi"
+!AddPluginDir /amd64-unicode   "../NSISPlugins/NScurl/Plugins/amd64-unicode"
```

Comment:
and are there ARM equivalents we need here?

Reply:
well shoot. No, not that NScurl releases. they don't build for arm in ci...

Reply:
Yeah, I'd definitely want to see this tested on a WoA machine (even if it has to virtualized) with them getting somewhat mainstream nowadays. This could be a bit nasty

Reply:
> Yeah, I'd definitely want to see this tested on a WoA machine (even if it has to virtualized) with them getting somewhat mainstream nowadays. This could be a bit nasty

https://discord.com/channels/1031648380885147709/1115661934478495835/1292718805801242687

We're go for arm install!

Reply:
> > Yeah, I'd definitely want to see this tested on a WoA machine (even if it has to virtualized) with them getting somewhat mainstream nowadays. This could be a bit nasty
> 
> https://discord.com/channels/1031648380885147709/1115661934478495835/1292718805801242687
> 
> We're go for arm install!